### ISSUE TITLE
Electron starts regardless of if a client is created

### ISSUE THREAD

**Issue by allanlw** [feross/webtorrent-hybrid#46]
The following program is expected to terminate, but it does not:

``` js
var WebTorrent = require("webtorrent-hybrid");

console.log("foo");
```

However, the following program does terminate immediately:

``` js
var WebTorrent = require("webtorrent");

console.log("foo");
```

This is because electron is started regardless of if a webtorrent client is created. Perhaps it could be lazy-initialized?

**Comment by DiegoRBaquero** [comment]
I've thrown two PR (one to the core lib and one to this module) to 'fix' this issue.

**Comment by feross** [comment]
Fixed in v1.0.5
